_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,191 @@
+</body>
+```
+
+#### Passed Example 2
+
+The text content automatically updates every 3 seconds after the page completes loading. A button is available to pause and resume the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="toggle()" value="Pause updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 3
+
+The text content automatically updates every 3 seconds after the page completes loading. A button is available to hide the automatically updating content.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="hide()" value="Hide updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The text content automatically updates every 3 seconds after the page completes loading. There is no component to stop or pause the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The text content automatically updates every 3 seconds after the page completes loading but it is not visible.
+
+```html
+<body onload="startUpdates()">
+	<p style="display: none">Random number: <span id="target">1</span></p>
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Inapplicable Example 2
+
+The text content automatically updates every 3 seconds but only as a result of a user interaction.
+
+```html
+<body>
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="startUpdates()" value="Start updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Inapplicable Example 3
+
+The automatically updating text content is the only content in the document.
```

Comment:
Technically, it's not the only content because there is also the button (and the "Random number:" text).

Reply:
👍